generators/structure/index.js

```diff
@@ -0,0 +1,36 @@
+const yeoman = require('yeoman-generator');
+const yosay = require('yosay');
+const _includes = require('lodash/includes');
+
+module.exports = yeoman.Base.extend({
+  prompting: {
+    structure() {
+      const self = this;
+
+      this.log(yosay(
+        'Create Folder Structure'
+      ));
+      return this.prompt([{
+        type: 'checkbox',
+        name: 'selectedProject',
```

Comment:
This should be `basicFolderStructure` and this should be a Boolean `true` or `false`.